src/text_engine/characters/PlayerCharacter.java

```diff
@@ -30,10 +27,17 @@
      */
     public PlayerCharacter(String name, String description, Room location, Item... inventory) {
         super(name, description, location, inventory);
+    }
+
+    @Override
+    protected ActionList actions(GameCharacter actor, GameEntity from, BufferedReader reader,
+                                 PrintWriter writer) {
+        ActionList actions = super.actions(actor, from, reader, writer);
+
+        actions.add(Actions.LOOK_AROUND);
+        actions.add(Actions.EXIT_GAME);
 
-        // Actions
-        actions.add(GameCharacter::getLocation);
-        actions.add(Interactive::exitGame);
+        return actions;
```

Comment:
Oooh! Aaah! A generic way to override actions without having to override the entire `respondToInteraction` method!